unattended.sh

```diff
@@ -1,178 +1,615 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
The is somehow limitating.
Bash is not always available, particularly on minimal installs.

But I agree that the script is much simpler for bash than it would be for sh.
So we choose simplicity of the script over the compatibility with more Linux installs?